src/main/java/com/sumologic/client/Credentials.java

```diff
@@ -7,37 +7,62 @@
  * @author Sebastian Mies
  */
 public class Credentials {
-    private String email;
-    private String password;
+    private String accessId;
+    private String accessKey;
 
     /**
-     * Constructs the credentials using email and password.
+     * Constructs the credentials using accessId and accessKey.
      *
-     * @param email Your email
-     * @param password Your password
+     * @param accessId Your access id
+     * @param accessKey Your access key
      */
-    public Credentials(String email, String password) {
-        this.email = email;
-        this.password = password;
+    public Credentials(String accessId, String accessKey) {
```

Comment:
We actually want to this behavior and keep the client code working without any change until we cut the support from server side. the accessId/key is interchangable with username/password in `Credentials`. In this way, user won't need to change their code to use accessId/accessKey. Also there are already a lot of places passing accessId/key to this constructor to create the client.